apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -114,7 +124,7 @@ export default connect(
     static propTypes = {
       // from redux
       commandHistory: PropTypes.instanceOf(CommandHistory),
-      logOutput: PropTypes.string.isRequired,
+      logOutput: flagOnLogOutputPropType(),
```

Comment:
This might be a good place to add some syntax magic and use a [ternary](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator). That way you don't need to create a whole new function.

Reply:
Brilliant! I'll go ahead and use ternary in other places too when it's a short if/else statement.